File: eRecipes/eRecipes.Model/FitPasos.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRecipes.Model
{
    public class FitPasos
    {
        public int FitPasosId { get; set; }
        public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }
        public DateTime DatumIzdavanja { get; set; }
        public bool Validan { get; set; }
    }
}

File: eRecipes/UI/erecipes_mobile/lib/models/fit_pasos.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:erecipes_mobile/models/korisnik.dart';


part 'fit_pasos.g.dart';

@JsonSerializable()
class FitPasos {
  int? fitPasosId;
 int? korisnikId;
 Korisnik? korisnik;
 DateTime? datumIzdavanja;
 bool? validan;

  FitPasos(
    this.korisnikId,
    this.datumIzdavanja,
    this.validan
  );

  factory FitPasos.FromJson(Map<String, dynamic> json) =>
      _$FitPasosFromJson(json);

  Map<String, dynamic> toJson() => _$FitPasosToJson(this);
}
